lib/systems/doubles.nix

```diff
@@ -91,6 +91,7 @@ in {
   gnu           = filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnu; })
                   ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnueabi; })
                   ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnueabihf; })
+                  ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabio32; })
```

Comment:
Adding this commit did not fix evaluation:
https://hercules-ci.com/github/nix-community/cross-toolchains.nix/jobs/80

Reply:
```
nix-repl .
nix-repl> :b pkgsCross.mipsisa32r6-linux-gnu.stdenv
error: Unknown CPU type: mipsisa32r6
```